frontend/containers/AddPassengerButtons/index.tsx

```diff
@@ -19,6 +19,10 @@ const AddPassengerButtons = ({
 }: Props) => {
   const theme = useTheme();
   const {t} = useTranslation();
+  const addToLocale = {
+    'waitingList': 'travel.passengers.add_to_waitingList',
+    'travel': 'travel.passengers.add_to_travel'
+  }
 
```

Comment:
@SimonMulquin This is a constant so we can put it before the component declaration with an uppercased name.